src/NUnitEngine/nunit.engine.tests/Services/ExtensionServiceTests.cs

```diff
@@ -256,8 +258,8 @@ public static IEnumerable<TestCaseData> InvalidTargetFrameworkCombos()
 
         public static IEnumerable<TestCaseData> InvalidRunnerCombos()
         {
-#if NETCOREAPP2_1
-            Assembly netstandard = typeof(ExtensionService).Assembly;
+#if NETCOREAPP
+            Assembly netstandard = Assembly.LoadFile(Path.Combine(GetSiblingDirectory("netstandard2.0"), "nunit.engine.dll"));
```

Comment:
These tests were using the executing engine as an example .NET Standard assembly - which of course caused problems when that assembly is build for .NET Core. I changed them to fetch the .NET Standard build specifically, in the same way the other tests in this class do.

Ideally, I think we need a set of test assemblies for this. (Or perhaps to use mock-assembly?)